frontend/src/ducks/transactions.ts

```diff
@@ -0,0 +1,91 @@
+import { createAsyncThunk, createSlice } from "@reduxjs/toolkit";
+
+import {
+  ledgerTransactionHistoryConfig,
+  networkConfig,
+} from "constants/settings";
+import {
+  TransactionsInitialState,
+  Network,
+  RejectMessage,
+  ActionStatus,
+  FetchTransactionsHistoryActionResponse,
+} from "types";
+import { RootState } from "config/store";
+import { getErrorString } from "helpers/getErrorString";
+import { parseDateFromFormat } from "helpers/parseDateFromFormat";
+
+export const fetchTransactionsHistoryAction = createAsyncThunk<
```

Comment:
this is just grabbing the last 30 day history, and since the backend also includes 24h and 1h I think we should label this as `month` as well.

maybe `fetchTransactionsHistoryMonthAction`?

Reply:
of course 👍🏽